torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -1,77 +1,82 @@
-from __future__ import annotations
```

Comment:
I would preserve `from __future__ import annotations` because it is what python will default to

Reply:
Thanks, this wasn't meant to be removed.